tools/ci/generate_sql_scripts.py

```diff
@@ -39,24 +44,36 @@
 scriptLines = [
     "#!/bin/bash\n",
     "set -euo pipefail\n"
+    "python3 -m pip install setuptools\n" # Yes I know you can PIP multiple things but they need to happen in this order
+    "python3 -m pip install mysql-connector\n"
     "mysql -u root -proot < tools/ci/sql_v0.sql\n"
 ]
 
 # And write the files and tell them to be used
 for file in orderedSqlFiles:
-    inFile = open("SQL/updates/" + file, "r")
-    fileLines = inFile.readlines()
-    inFile.close()
-    # Add in a line which tells it to use the feedback DB
-    fileLines.insert(0, "USE `feedback`;\n")
+    if file.endswith(".py"):
+        # Begin snowflakery
+        if file == "16-17.py":
+            scriptLines.append("python3 SQL/updates/" + str(file) + " 127.0.0.1 root root feedback feedback round\n")
+        elif file == "17-18.py":
+            scriptLines.append("python3 SQL/updates/" + str(file) + " 127.0.0.1 root root feedback feedback feedback_2\n")
+        else:
+            print("ERROR: CI failed due to invalid python file in SQL/updates")
+            exit(1)
+    else:
+        inFile = open("SQL/updates/" + file, "r")
```

Comment:
honestly at this point `SQL/updates/file` should be its own variable given that you've used it 3 times.
And btw the canonical way of making the full path would be `os.path.join("SQL", "updates", file)` (or if you're lazy, `os.path.join("SQL/updates", file)` is a middle-ground that happens to work (same as `+`), but also feels good because it's the "proper" way).

Reply:
Ditto